### PR TITLE
Version flag

### LINKED ISSUE #47: Add --version flag
Add `--version` flag, which should returns the version of installed version of squash

### PR DESCRIPTION
**Pull request by ArthurCueio**
Fixes #47 

Check everything which applies

- [x] I have added the issue number for which this pull request is created.

### CODE REVIEW

File: lib/util/print-version.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const printVersion = () => {
+  let pjson = require('../../package.json');
```
Comment (arshadkazmi42) [inline lib/util/print-version.js]: Lets move this import to top of the file. 
So all imports will be at one place.
Also, let's change this variable name `pjson`, to something like `PACKAGE_CONFIG` or some similar name

File: lib/util/print-version.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  console.log(`Squash version is ${pjson.version}`);
```
Comment (arshadkazmi42) [inline lib/util/print-version.js]: We want to print just the version squash without anything else.
Let's change this to `console.log(pjson.version)`

File: tests/test.js [hunk 3]
```diff
@@ -121,4 +123,23 @@ describe('Verifies async features', function() {
...
+    expect(console.log.getCall(6).args[0]).to.equal('####       ####  ########  ###  ########  ###  ########  #####         ####            ##');
+    expect(console.log.getCall(7).args[0]).to.equal('#########   ###  ########  ###  ########  ###            ############  ####  ########  ##');
+    expect(console.log.getCall(8).args[0]).to.equal('##########  ####  ######  ####  ########  ###  ########  #############  ###  ########  ##');
+    expect(console.log.getCall(9).args[0]).to.equal('#########  ######       #######  ######  ####  ########  ############  ####  ########  ##');
+    expect(console.log.getCall(10).args[0]).to.equal('###       #############  ########      ######  ########  ####        ######  ########  ##');
+    expect(console.log.getCall(11).args[0]).to.equal('#########################################################################################');
+    expect(console.log.getCall(12).args[0]).to.equal('\n');
+    expect(console.log.getCall(13).args[0]).to.contain('Squash version is '); // We don't want to check for the actual version number to avoid changing this every release
```
Comment (arshadkazmi42) [inline tests/test.js]: This needs to be modified based on above changes.
For dynamic versions check, you can import package.json in the test file and then can match the version with the printed version

**Comment by ArthurCueio** [comment]
Hey @arshadkazmi42 I made the changes you requested. :+1:

**Pull request merged**